src/Controls/src/Core/Handlers/Items/Android/ItemContentView.cs

```diff
@@ -83,26 +83,39 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 				return;
 			}
 
-			if (_size != null)
+			// If we're using ItemSizingStrategy.MeasureFirstItem and now we have a set size, use that
+			// Even though we already know the size we still need to pass the measure through to the children.
+			if (_size is not null)
 			{
-				// If we're using ItemSizingStrategy.MeasureFirstItem and now we have a set size, use that
-				SetMeasuredDimension((int)_size.Value.Width, (int)_size.Value.Height);
+				var w = (int)_size.Value.Width;
+				var h = (int)_size.Value.Height;
+
+				// If the platform childs measure has been invalidated, it's going to still want to
+				// participate in the measure lifecycle in order to update its internal
+				// book keeping.
+				_ = View.Measure
+					(
+						MeasureSpec.MakeMeasureSpec(w, MeasureSpecMode.Exactly),
+						MeasureSpec.MakeMeasureSpec(h, MeasureSpecMode.Exactly)
+					);
+
+				SetMeasuredDimension(w, h);
 				return;
 			}
 
 			int pixelWidth = MeasureSpec.GetSize(widthMeasureSpec);
 			int pixelHeight = MeasureSpec.GetSize(heightMeasureSpec);
 
-			var width = MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.Unspecified
+			var widthSpec = MeasureSpec.GetMode(widthMeasureSpec) == MeasureSpecMode.Unspecified
```

Comment:
I think this was correct before. This code is using the measure spec to get the pixel width and then convert to independent width. The spec was the source value, this is now just the width in dip.

Reply:
I don't think there's a behavior change here
I was just renaming things